README.md

```diff
@@ -1,16 +1,61 @@
 # Boreholes Data Extraction
 
-Boreholes Data Extraction is a data extraction pipeline that extracts depth layers with their corresponding material description from borehole profiles in form of pdfs.
+Boreholes Data Extraction is a pipeline to extract structured data from borehole profiles in PDF files. Extracted properties (currently coordinates, depths and associated material descriptions) are returned in JSON format, and (optionally) visualized as PNG images. This project was initiated by the Swiss Federal Office of Topography [swisstopo](https://www.swisstopo.admin.ch/), and is developed with support from [Visium](https://www.visium.ch/).
 
-## Limitations
+## Motivation
 
-Note that the project is under active development and there is no release to this date, nor has the project reached a maturity such that it could be used.
+The Federal Office of Topography swisstopo is Switzerland's geoinformation centre. The Swiss Geological Survey at swisstopo is the federal competence centre for the collection, analysis, storage and provision of geological data of national interest. 
```

Comment:
Missing comma: 

The Federal Office of Topography swisstopo is Switzerland's geoinformation centre. The Swiss Geological Survey at swisstopo is the federal competence centre for the collection, analysis, storage, and provision of geological data of national interest.

Reply:
That's not a universally accepted rule :) -> https://en.wikipedia.org/wiki/Serial_comma
However, I don't feel very strongly about it, and I can add the comma.

A general note: could you try using the "add a suggestion" Github feature for suggesting minor changes like this one? That would make it easier to find and to accept the suggested changes.
![image](https://github.com/user-attachments/assets/0ed15214-96bf-48d9-bfb0-2d31691aa6a6)